website/docs/StaticFiles.md

```diff
@@ -34,4 +34,35 @@ You can create a file at `.moonwave/custom.css`. Here's an example of what you c
   --ifm-color-primary-lightest: rgb(146, 224, 208);
   --ifm-code-font-size: 95%;
 }
-```
\ No newline at end of file
+```
+
+## Custom Docs Sidebar
+
+You can create a custom `.moonwave/sidebars.js` file to set up a custom ordering for your `docs` sidebar. This allows you to specify ordering, sections, and if certain files are excluded.
+
+Example `sidebars.js` file:
+
+```js
+module.exports = {
+  mySidebar: [
+    {
+        type: "doc",
+        id: "getting-started"
+        label: "Getting Started"
+    },
+    {
+      type: 'category',
+      label: 'Moonwave',
+      items: ['moonwave-basics, moonwave-advances'],
+    },
+    {
+      type: 'category',
+      label: 'Other Resources',
+      items: ['nested-folder/extra-resources', 'another-folder/even-more-resources'],
+    },
+
+  ],
```

Comment:
Can you format this snippet with prettier in a standalone javascript file and then paste it back here?